### ISSUE TITLE
Simplify wallet registration UX flow

### ISSUE THREAD

**Issue by bigspider** [LedgerHQ/app-bitcoin-new#253]
After #252, we could consider rewording the screens in the wallet registration so that it says "Continue" instead of "Approve", except for the final step, which is actually the user approval that matters.

If possible, it would be even better to completely remove all the intermediate button presses: the user just keeps scrolling right until "Approve"/"Reject" is shown after inspecting the last key.

**Comment by bigspider** [comment]
Mostly done in #270; only partially for devices not using NBGL - but it should come for free once all devices migrate to NBGL.

**Comment by bigspider** [comment]
No longer relevant.
